packages/loader/driver-utils/src/buildSnapshotTree.ts

```diff
@@ -14,26 +14,27 @@ import {
     TreeEntry,
 } from "@fluidframework/protocol-definitions";
 import { buildHierarchy } from "@fluidframework/protocol-base";
+import { v4 as uuid } from "uuid";
 
-async function flattenCore(
+function flattenCore(
     path: string,
     treeEntries: ITreeEntry[],
     blobMap: Map<string, string>,
-): Promise<git.ITreeEntry[]> {
+): git.ITreeEntry[] {
     const entries: git.ITreeEntry[] = [];
     for (const treeEntry of treeEntries) {
         const subPath = `${path}${treeEntry.path}`;
 
         if (treeEntry.type === TreeEntry.Blob) {
             const blob = treeEntry.value as IBlob;
             const buffer = IsoBuffer.from(blob.contents, blob.encoding);
-            const sha = await gitHashFile(buffer);
-            blobMap.set(sha, buffer.toString("base64"));
+            const id = uuid();
+            blobMap.set(id, buffer.toString("base64"));
 
             const entry: git.ITreeEntry = {
                 mode: FileMode[treeEntry.mode],
                 path: subPath,
-                sha,
+                sha: id,
```

Comment:
How hard is it to remove git lingvo from here? Using git interfaces here is very confusing and is not needed, as far as I can tell

Reply:
I agree. i briefly looked at it, but quickly ended up in protocol definitions which is harder to stage. I'll file an issue as i agree we shouldn't need git on the on client side beyond maybe a driver that depends on it.